crates/sargon-uniffi/src/keys_collector/key_derivation_response.rs

```diff
@@ -0,0 +1,47 @@
+use crate::prelude::*;
+use sargon::IndexMap;
+use sargon::IndexSet;
+use sargon::KeyDerivationResponse as InternalKeyDerivationResponse;
+
+#[derive(Clone, PartialEq, Eq, uniffi::Record)]
+pub struct KeyDerivationResponse {
```

Comment:
doc

Reply:
Aaaah forgot to change the HashMap in `KeyDerivationResponse`